terragrunt/aws/software_asset_inventory/efs.tf

```diff
@@ -8,6 +8,14 @@ resource "aws_efs_file_system" "dependencytrack" {
   }
 }
 
+resource "aws_efs_backup_policy" "policy" {
```

Comment:
Same commend about name - could you name this something like so to give us unique resource names:
```hcl
resource "aws_efs_backup_policy" "dependencytrack"
```